Source/WebCore/css/CSSFontFaceSet.cpp

```diff
@@ -396,10 +396,10 @@ ExceptionOr<Vector<std::reference_wrapper<CSSFontFace>>> CSSFontFaceSet::matchin
                 familyAtom = familyNamesData->at(CSSPropertyParserHelpers::genericFontFamilyIndex(familyKeyword));
             else {
                 ASSERT(m_owningFontSelector && m_owningFontSelector->scriptExecutionContext());
-                familyAtom = m_owningFontSelector->scriptExecutionContext()->settingsValues().fontGenericFamilies.standardFontFamily();
+                familyAtom = AtomString { m_owningFontSelector->scriptExecutionContext()->settingsValues().fontGenericFamilies.standardFontFamily() };
```

Comment:
Can we change fontGenericFamilies to use `AtomString`?

Reply:
I think so, I'll try.

Reply:
Hmm, FontGenericFamilies is passed across thread and has an isolatedCopy() function. This makes using AtomString not practical.